### DISCUSSION TITLE
Export Pathing as File Paths

### DISCUSSION THREAD

**Discussion by kalsprite** [webpack/webpack#13264] [Q&A]
I am looking to run a webpage from the file-system using WebView2. webpack exports the relative pathing as `/css/file.css`, however since I am serving from a folder, i need to export it as `./css/file.css`. 

Is there a setting that would let me do this?

**Comment by alexander-akait** [comment]
Do you use `mini-css-extract-plugin`?

**Reply by kalsprite** [reply]
Webpack is buried in Vue in my current project, per the CLI docs it does appear it uses [mini-css](https://cli.vuejs.org/config/#css-extract). Is that related? 

At the end of the day I am trying to output file:/// compatible relative paths instead of web.

**Reply by alexander-akait** [reply]
You need specify `publicPath: ''` for `mini-css-extract-plugin` in this case

**Reply by kalsprite** [reply]
does not change the output:
output index.html is:
`<script src="js/chunk-vendors.js">`

need output file to be:
`<script src="./js/chunk-vendors.js">`

**Reply by alexander-akait** [reply]
`publicPath: 'auto'`, but you need `webpack` v5 for this
